mbed_tools_lib/logging.py

```diff
@@ -0,0 +1,34 @@
+"""Helpers for logging errors according to severity of the exception."""
+import logging
+from typing import Any
+
+LOGGING_FORMAT = "%(levelname)s: %(message)s"
+
+
+def log_exception(logger: Any, exception: Exception) -> None:
+    """Logs an exception in both normal and verbose forms.
+
+    Args:
+        logger: logger
+        exception: exception to log
+    """
+    if logger and exception:
```

Comment:
If they are no longer optional, do we need this conditional?

Reply:
Good point, I don't see why we would.